omnibus/package-scripts/updater-deb/postinst

```diff
@@ -99,9 +101,9 @@ set -e
 # Set proper rights to the dd-agent user
 chown -R dd-agent:dd-agent ${CONFIG_DIR}
 chown -R dd-agent:dd-agent ${LOG_DIR}
```

Comment:
Should there be an additional `chmod` to ensure those 2 folders are writable by the group?

Reply:
yes, that's better then the log chmod below, updating